src/renderers/art/ReactART.js

```diff
@@ -0,0 +1,643 @@
+/**
+ * Copyright (c) 2013-present Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ReactARTFifteen
```

Comment:
Could always just not copy this whole directory to `lib/` (which we probably want to do for the noop renderer from #6690 too)

```diff
--- a/gulpfile.js
+++ b/gulpfile.js
@@ -20,6 +20,7 @@ var paths = {
   react: {
     src: [
       'src/**/*.js',
+      '!src/renders/art/**/*.js',
       '!src/**/__benchmarks__/**/*.js',
       '!src/**/__tests__/**/*.js',
       '!src/**/__mocks__/**/*.js',
```